app/jobs/build_suggest_job.rb

```diff
@@ -2,10 +2,12 @@
 
 # Update the Spellcheck database to enable autocomplete
 class BuildSuggestJob < ApplicationJob
-  def perform
-    Search.client.send_and_receive(
-      'suggest',
-      params: { 'suggest.build' => true }
-    )
+  discard_on RSolr::Error::Http
```

Comment:
We just don't care about the response here, so let's discard it. Unfortunately we will likely never get a 200 because Solr's apache will timeout.